### PR TITLE
Added some code to increase coverage

### PR DESCRIPTION
**Pull request by RidhimaKohli**
Code didn't raise error when heap property was something other than max and min

### CODE REVIEW

**Comment by HarsheetKakar** [comment]
Test is already there 
```
def test_DHeap():
    assert raises(ValueError, lambda: DHeap(heap_property="none", d=4))
```

File: pydatastructs/trees/tests/test_heaps.py [hunk 1]
```diff
@@ -93,6 +93,9 @@ def test_TernaryHeap():
     expected_sorted_elements = [2, 3, 7, 17, 19, 25, 36, 100]
     sorted_elements = [min_heap.extract().key for _ in range(8)]
     assert expected_sorted_elements == sorted_elements
+<<<<<<< HEAD
+    raises(ValueError,lambda:BinaryHeap(heap_property='arbitrary'))
+=======
```
Comment (czgdp1807) [inline pydatastructs/trees/tests/test_heaps.py]:
```suggestion
    raises(ValueError,lambda:BinaryHeap(heap_property='arbitrary'))
```

File: pydatastructs/trees/tests/test_heaps.py [hunk 2]
```diff
@@ -133,6 +134,7 @@ def test_DHeap():
     sorted_elements = [min_heap.extract().key for _ in range(8)]
     assert expected_sorted_elements == sorted_elements
 
+>>>>>>> 835aeef937c270062b713ad47b39b1788d69b374
```
Comment (czgdp1807) [inline pydatastructs/trees/tests/test_heaps.py]:
```suggestion
```

**Comment by czgdp1807** [comment]
Closing as the purpose of the PR is increasing coverage which is not reflected in [this comment](https://github.com/codezonediitj/pydatastructs/pull/167#issuecomment-600035483).
Thanks for the contributions. Please see, [testing guide](https://github.com/codezonediitj/pydatastructs#testing) for analysing the coverage locally in your system. The lines marked in red are those which aren't executed even once by the tests and tests should be added for these lines.

**Pull request closed without merging**